tox.ini

```diff
@@ -1,27 +1,46 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3,docs
+    py27,
+    py33,
+    py34,
+    py35,
+    py36,
+    pypy,
+    pypy3,
+    docs,
+    coverage-report,
 
 [testenv]
+usedevelop = true
```

Comment:
Thank you for explaining!

You could also solve that by having a `.coveragerc` with
```
[paths]
source =
    src/zope/authentication
    .tox/*/lib/python*/site-packages/zope/authentication
    .tox/pypy/site-packages/zope/authentication
```
but `usedevelop` is certainly simpler.